### PR TITLE
POL-887 Meta Policy for AWS Lambda Function high error rate

### PR DESCRIPTION
**Pull request by nia-vf1**
### Description

This is a change that adds meta policy functionality to the AWS Lambda Function High Error Rate policy template.

### Issues Resolved

Resolves issue in all customer tenants where the Lambda Function High Error Rate policy must be ran individually on each linked account and master account.

### Link to Example Applied Policy

https://app.flexera.com/orgs/30105/automation/applied-policies/projects/133807?policyId=64bfb4b6fbdef500017291e6 

### Contribution Check List

- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/aws/lambda_functions_with_high_error_rate/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v4.0
+
+- Added logic required for "Meta Policy" use-cases
+- Updated template and readme to reflect addition of `auth_flexera`
```
Comment (XOmniverse) [inline operational/aws/lambda_functions_with_high_error_rate/CHANGELOG.md:6]: I would change this to something more human readable to a non-technical user. Something like:

- Flexera credential now required to facilitate meta policy use cases.

**Review by XOmniverse — CHANGES_REQUESTED** [review]
One minor change. Otherwise it looks good :)

File: operational/aws/lambda_functions_with_high_error_rate/CHANGELOG.md (hunk 1 above)
Comment (nia-vf1) [reply operational/aws/lambda_functions_with_high_error_rate/CHANGELOG.md]: Done

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**